receivers/line/config.go

```diff
@@ -0,0 +1,36 @@
+package line
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/grafana/alerting/receivers"
+	"github.com/grafana/alerting/templates"
+)
+
+type Config struct {
+	Token       string `json:"token,omitempty" yaml:"token,omitempty"`
+	Title       string `json:"title,omitempty" yaml:"title,omitempty"`
+	Description string `json:"description,omitempty" yaml:"description,omitempty"`
+}
+
+func BuildConfig(fc receivers.FactoryConfig) (*Config, error) {
```

Comment:
nit: This is validating the configuration but it's called `BuildConfig` in all receivers - this feels misleading, I understand that this "technically" returning a configuration because it's doing the marshalling but the common pattern I see in most cases is to simply execute the unmarshall call inline and call `Validate()` on top of the built object.

Reply:
I do not have a strong opinion on this name. BuildConfig and Validate have pros and cons. Ideally, that should be boiled down to custom marshaller implementation for each struct but for now, decryption functionality does not let us do that (I have a ddoc for that).

Reply:
I renamed BuildConfig to ValidateConfig.